app/assets/javascripts/Components/Result/result.jsx

```diff
@@ -141,8 +141,12 @@ class Result extends React.Component {
         }];
       } else {
         children = annotation_category.texts.map(text => {
+          let deduction = '';
+          if (text.deduction !== undefined && text.deduction !== null && text.deduction !== 0) {
+            deduction = '-' + text.deduction;
+          }
           return {
-            title: text.content.replace(/\r?\n/gi, ' '),
+            title: `${text.content.replace(/\r?\n/gi, ' ')} <span class="red-text">${deduction}</span>`,
```

Comment:
Perhaps add `text-content` class to title text so deduction is to the right of the element.

Reply:
No effect, and hard to debug due to content menu disappearing when inspecting element.